app/javascript/src/Settings/components/Common/TextInputGroup.jsx

```diff
@@ -0,0 +1,38 @@
+// @flow
+
+import * as React from 'react'
+import { useState } from 'react'
+import { FormGroup, TextInput } from '@patternfly/react-core'
+import type { FieldGroupProps } from 'Settings/types'
+
+const TextInputGroup = ({
+  defaultValue,
+  placeholder,
+  label,
+  name,
+  hint,
+  value,
+  isDefaultValue = false,
+  readOnly = false,
+  inputType = 'text'
+}: FieldGroupProps) => {
+  const [ inputValue, setInputValue ] = useState(value)
+  const onChange = (value, _e) => setInputValue(value)
```

Comment:
```suggestion
  const onChange = setInputValue
```

Reply:
As I commented in https://github.com/3scale/porta/pull/1315#discussion_r340999672, I can't use point free notation here because of React annoying warning.